lib/FusionInventory/Agent/Task/Inventory/Win32/Softwares.pm

```diff
@@ -316,4 +339,55 @@ sub _processMSIE {
 
 }
 
+# List of SQL Instances
+sub _getSqlInstancesList {
+    my (%params) = @_;
+
+    my $softwareName = $params{softwarename};
+    my $softwareVersion = $params{softwareversion};
+
+    # Registry access for SQL Instances
+    my $sqlinstancesList = getRegistryKey(
+        path    => "HKEY_LOCAL_MACHINE/SOFTWARE/Microsoft/Microsoft SQL Server/Instance Names/SQL",
+        %params
+    ) or return;
```

Comment:
this pattern is not consistent with current coding style, and should rather get turned into:
return unless $sqlinstancesList;